crates/uv/tests/edit.rs

```diff
@@ -763,6 +931,122 @@ fn add_remove_workspace() -> Result<()> {
     Ok(())
 }
 
+/// Add a workspace dependency as an editable.
+#[test]
+fn add_workspace_editable() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let workspace = context.temp_dir.child("pyproject.toml");
+    workspace.write_str(indoc! {r#"
+        [tool.uv.workspace]
+        members = ["child1", "child2"]
+    "#})?;
+
+    let pyproject_toml = context.temp_dir.child("child1/pyproject.toml");
+    pyproject_toml.write_str(indoc! {r#"
+        [project]
+        name = "child1"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = []
+    "#})?;
+
+    let pyproject_toml = context.temp_dir.child("child2/pyproject.toml");
+    pyproject_toml.write_str(indoc! {r#"
+        [project]
+        name = "child2"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = []
+    "#})?;
+
+    let child1 = context.temp_dir.join("child1");
+    let mut add_cmd = context.add(&["child2"]);
+    add_cmd
+        .arg("--editable=true")
```

Comment:
I'm not sure how to make `--editable` a flag with it being represented as an `Option<bool>`. It's optional because we don't want to write `editable = true/false` if it's not provided as that's implied.

Reply:
I think you're looking for [`default_missing_value`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.default_missing_value)